event-handler/teleport_events_watcher_test.go

```diff
@@ -108,3 +109,56 @@ func TestNext(t *testing.T) {
 		require.Fail(t, "No events were sent")
 	}
 }
+
+func TestValidateConfig(t *testing.T) {
+	for _, tc := range []struct {
+		name      string
+		cfg       StartCmdConfig
+		wantError bool
+	}{
+		{
+			name: "Identity and teleport cert/ca/key files configured",
+			cfg: StartCmdConfig{
+				FluentdConfig{},
+				TeleportConfig{
+					TeleportIdentityFile: "not_empty_string",
+					TeleportCA:           "not_empty_string",
+					TeleportCert:         "not_empty_string",
+					TeleportKey:          "not_empty_string",
```

Comment:
Let's make identity file mutually exclusive with CA/Cert/Key, like in the `CheckAndSetDefaults` example.